production/helm/promtail/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: "v1"
 name: promtail
-version: 0.16.1
+version: 0.17.0
```

Comment:
Because it is not a breaking or considerably big change why not bump version to `0.16.2` instead?